tests/phpunit/tests/rest-api/rest-attachments-controller.php

```diff
@@ -1045,6 +1045,24 @@ public function test_create_item_ensure_relative_path() {
 		$this->assertStringNotContainsString( ABSPATH, get_post_meta( $attachment['id'], '_wp_attached_file', true ) );
 	}
 
+	/**
+	 * @requires function imagejpeg
```

Comment:
```suggestion
	 * @ticket 57897
	 *
	 * @requires function imagejpeg
```

Add ticket number for the test.